opencanary/logger.py

```diff
@@ -144,12 +148,24 @@ def __init__(self, config, handlers, formatters={}, per_service_logs={}):
             # initialise all defined logger handlers
             "loggers": {self.node_id: {"handlers": set(handlers.keys())}},
         }
-        for psl in per_service_logs:
-            logconfig["handlers"][psl] = {
+
+        default_service_log_dir = (
+            os.environ.get(self.DEFAULT_SERVICE_LOG_DIR_VAR) or self.DEFAULT_SERVICE_LOG_DIR)
```

Comment:
slightly nicer to use `os.getenv()` which takes an arg for default: https://www.geeksforgeeks.org/python-os-getenv-method/

Reply:
Done